libs/input-mapping/tests/AbstractTestCase.php

```diff
@@ -64,11 +72,20 @@ public function setUp(): void
         }
     }
 
-    protected function initClient(): void
+    protected function initClient(?string $branchId = null): void
     {
-        $this->clientWrapper = new ClientWrapper(
-            new ClientOptions((string) getenv('STORAGE_API_URL'), (string) getenv('STORAGE_API_TOKEN')),
-        );
+        $clientOptions = (new ClientOptions())
+            ->setUrl((string) getenv('STORAGE_API_URL'))
+            ->setToken((string) getenv('STORAGE_API_TOKEN'))
+            ->setBranchId($branchId)
+            ->setBackoffMaxTries(1)
+            ->setJobPollRetryDelay(function () {
+                return 1;
+            })
+            ->setUserAgent(implode('::', Test::describe($this)))
+        ;
+
+        $this->clientWrapper = new ClientWrapper($clientOptions);
```

Comment:
tohle je nepraktický, radši ať je to getClient (nebo klidně i initClient), ale zbavil bych se toho $this->clientWrapper, to dělá akorát bordel